docs/content/walkthrough.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Walkthrough
+description: Understanding the Athens proxy and Go Modules
+menu: shortcuts
+---
+
+First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/) and that GOPATH/bin is on your path.
+
+## Without the proxy
+Let's review what everything looks like in Go 1.11 without the proxy in the picture:
+
+```console
+$ git clone https://github.com/athens-artifacts/walkthrough.git
+$ cd walkthrough
+$ GO111MODULE=on go run .
+go: downloading github.com/athens-artifacts/samplelib v1.0.0
+The 🦁 says rawr!
+```
+
+The end result of running this command is that Go downloaded the package source and packaged
+it into a module, saving it in the Go Modules cache.
+
+Now that we know what stuff looks like without the proxy, only using Go Modules, let's take a look at
+how the proxy changes what we see.
+
+## With the proxy
+Using the most simple installation possible, let's walk through how to use the
+Athens proxy, and figure out what is happening at each step. 
+
+Before moving on, let's clear our Go Modules cache so that we can see the proxy
+in action without any caches populated:
+
+**Bash**
+```bash
+sudo rm -fr $(go env GOPATH)/pkg/mod
+```
+
+**PowerShell**
+```powershell
+rm -recurse $(go env GOPATH)\pkg\mod
```

Comment:
This should be:
```
rm "$(go env GOPATH)\pkg\mod" -recurse -force
```

Reply:
Or your ordering, just needs to have `-force` because the the directories aren't empty, so it will fail without the `-force` flag.

Reply:
Thanks what you suggested is good, I was going off the top of my head on the syntax. 😀